doc-content/enterprise-only/project-data/delete-branches-proc.adoc

```diff
@@ -7,6 +7,6 @@ You can delete any branch except for the *master* branch. {CENTRAL} does not all
 . Click image:project-data/dots.png[] in the upper-right corner of the screen and select *Delete Branch*.
 +
 .Deleting a branch menu
-image::getting-started/delete-branch-menu.png[Delete a branch menu]
+image::getting-started/delete-branch-option.png[Delete a branch menu]
 
 . Input the branch name to be deleted in the *Delete Branch* window and click *Delete Branch*.
```

Comment:
Minor suggestion - confirm that "to be" is ok. I see we use more instances of "you want".

For example:
. "In the *Delete Branch* window, enter the name of the branch you want to delete."
. "Click *Delete Branch*.